package.json

```diff
@@ -106,12 +108,18 @@
   "scripts": {
     "build": "gatsby build",
     "develop": "gatsby develop",
-    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
+    "format": "prettier --write \"*/**/*.{js,jsx,ts,tsx,json,css,scss}\"",
```

Comment:
+1 to changing which extensions

but why change from `**/` to `*/**/` here? by adding a level it stops checking the top-level files which I don't think is intended?